resources/videos/README.md

```diff
@@ -0,0 +1,58 @@
+# Resources for making videos
+
+Here you will find resources for making videos for community management - promotion, training, translation of technical science (explainers), presentations. 
+
+## Links to the Turing Way book
+* Making videos for presentations - https://the-turing-way.netlify.app/communication/presentations/presentations-remote.html
+
+
+## How to make videos
+
+### Using Turing comms team
+* Contact Turing comms team to get professional video production - contact is Dan Whitfield or Sian Fortt. 
+
+### Making your own video - general guidance
+* See the [recording tips guide from ELIXIR-UK](https://github.com/alan-turing-institute/open-community-building/blob/video-resources-docs/resources/videos/recording-tips.md)
+
+### Recording tools/software
+* Link to [SSI fellowship video recording guide](https://www.software.ac.uk/fellowship-programme/2022/application-video-guide) - using OBS studio and Zoom.
+* [Recording in powerpoint](https://github.com/alan-turing-institute/open-community-building/blob/video-resources-docs/resources/videos/how-to-record-powerpoint-windows.pdf) - by ELIXIR-UK. 
+
```

Comment:
[Adobe Express](https://www.adobe.com/express/feature/video/editor) is a free video editor that lets you add some simple effects. However you can only edit the video in 30 second intervals so this will not be useful for long videos!